BackendApartmentReservation/BackendApartmentReservation/Infrastructure/Exceptions/ErrorCodes.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BackendApartmentReservation.Infrastructure.Exceptions
+{
+    public static class ErrorCodes
+    {
+        public static string GenericInternalServerError = "generic.internal_error";

```

Comment:
**Comment**
A static field that is neither constant nor read-only is not thread-safe. Recommended `const` or `readonly`. Info from Sonar.

Reply:
Oh, yes, const really makes sense here. Will add